mantle/cmd/kola/testiso.go

```diff
@@ -250,13 +250,28 @@ wait-device-timeout=20000
 method=auto
 `, nmConnectionId)
 
+var nmstateConfigFile = "/etc/nmstate/br-ex.yml"
+var nmstateConfig = `interfaces:
+ - name: br-ex
+   type: linux-bridge
+   state: up
+   ipv4:
+     enabled: false
+   ipv6:
+     enabled: false
+   bridge:
+     port: []
+`
+
 // This is used to verify *both* the live and the target system in the `--add-nm-keyfile` path.
 var verifyNmKeyfile = fmt.Sprintf(`[Unit]
 Description=TestISO Verify NM Keyfile Propagation
 OnFailure=emergency.target
 OnFailureJobMode=isolate
 Wants=network-online.target
 After=network-online.target
+# XXX - should be part of network-online.target?
+After=nmstate.service
```

Comment:
This unit already waits for `network-online.target`, but that isn't good enough here, since NetworkManager successfully DHCPs and reports ready before `nmstate.service` finishes running.  Should `nmstate.service` add itself as `Before=network-online.target`?

Reply:
Notably, [coreos-installer.service](https://github.com/coreos/coreos-installer/blob/main/systemd/coreos-installer.service) waits for `network-online.target`.  Without this, it would presumably have to hardcode `nmstate.service` as well.

Reply:
Fixed upstream in https://github.com/nmstate/nmstate/pull/2286.